tailwind.config.js

```diff
@@ -28,17 +28,42 @@ module.exports = {
           "base-200": "#f1f5f9",
           "base-300": "#e2e8f0",
 
-          primary: "#4885ed",
-          "primary-content": "#E8F0FE",
```

Comment:
We need add back and define all the `-content` colors as well. If Harish hasn't defined these, work with him to get it done.

See here: https://daisyui.com/docs/colors/#-2